generate-load.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
```

Comment:
I ran out of ddosify credits and didn't want to risk anything for the presentation